agent/csi/libstorage/csi_service_libstorage.go

```diff
@@ -448,6 +459,11 @@ func (d *driver) NodePublishVolume(
 	target := req.GetTargetPath()
 	vi := req.GetPublishVolumeInfo()
 
+	st, err := os.Stat(target)
```

Comment:
Hi @codenrhoden,

We already import gotil, so why not use it? Just curious.

Reply:
yep, exactly. I started with `gotil` as requested, but then noticed I was going to use `os.Stat` to find out if it is a dir anyways...